glide.yaml

```diff
@@ -70,3 +70,13 @@ import:
   version: 335da9dda11408a34b64344f82e9c03779b71673
   repo: https://github.com/openshift/glog.git
   vcs: git
+- package: github.com/stretchr/testify
```

Comment:
Right, that's how you would do it. I also would like to have another test framework, but I am reluctant to just pick this one w/o any research. Check gocheck - http://labix.org/gocheck - for example. It also offers assertions, but also text fixtures, something I dearly miss as well. So why not gocheck? Have you had a look at it? If we want this merged quickly, I suggest we stick to the more cumbersome assertions and leave test framework investigations for later. Feel free to provide a list of possibilities, but I feel more research is required here.

I am also a bit confused about the rather large changeset in the Glide lock file. To merge this pull request I would like to run some experiments there first which is also holding up a quick merge.